docs/build/guides/transactions/pooled-accounts-muxed-accounts-memos.mdx

```diff
@@ -116,146 +247,262 @@ async function preamble() {
 
-```
-Custodian:
-       GCIHAQVWZH2AB5BB5NP63FBSIREG77LQZZNUVKD2LN2IOCLOT6N72MJN
-Customers:
-    1: MCIHAQVWZH2AB5BB5NP63FBSIREG77LQZZNUVKD2LN2IOCLOT6N72AAAAAAAAAAAAEDB4
-   22: MCIHAQVWZH2AB5BB5NP63FBSIREG77LQZZNUVKD2LN2IOCLOT6N72AAAAAAAAAAAC3IHY
-  333: MCIHAQVWZH2AB5BB5NP63FBSIREG77LQZZNUVKD2LN2IOCLOT6N72AAAAAAAAAABJV72I
- 4444: MCIHAQVWZH2AB5BB5NP63FBSIREG77LQZZNUVKD2LN2IOCLOT6N72AAAAAAAAAARLQOKK
-```
-
-With the accounts out of the way, let’s look at how we can manage the difference between traditional Stellar accounts (G...) and these virtual muxed accounts (M...).
+### Example 1 - Basic Payment from G account to G account
 
-#### Muxed Operations Model
-
-The introduction of muxed addresses as a higher-level abstraction—and their experimental, opt-in nature—means there are mildly diverging branches of code depending on whether the source is a muxed account or not. We still need to, for example, load accounts by their underlying address, because the muxed versions don’t actually live on the Stellar ledger:
+This example demonstrates a standard payment between two regular Stellar accounts (G... addresses) to establish the baseline behavior. Both the custodian and outsider accounts exist directly on the Stellar ledger, so this payment follows the traditional Stellar payment flow without any muxed account abstractions.
 
 <CodeExample>
 
 ```js
-function loadAccount(account) {
-  if (StellarSdk.StrKey.isValidMed25519Address(account.accountId())) {
-    return loadAccount(account.baseAccount());
-  } else {
-    return server.loadAccount(account.accountId());
-  }
-}
+async function runUnmuxedExample() {
+  try {
+    await preamble();
+
+    console.log("=== INITIAL BALANCES ===");
+    await showBalance(custodianAcc);
+    await showBalance(outsiderAcc);
+    console.log();
+
+    await makePayment(
+      custodianAcc,
+      outsiderAcc,
+      "Basic Payment from G to G address",
+    );
 
-function showBalance(acc) {
-  console.log(`${acc.accountId().substring(0, 5)}: ${acc.balances[0].balance}`);
+    console.log("\n=== FINAL BALANCES ===");
+    const finalCustodianAcc = await server.getAccount(custodian.publicKey());
+    const finalOutsiderAcc = await server.getAccount(outsider.publicKey());
+    await showBalance(finalCustodianAcc);
+    await showBalance(finalOutsiderAcc);
+
+    console.log("\n=== BASIC PAYMENT EXAMPLE COMPLETED ===");
+  } catch (error) {
+    console.error("Error in example:", error.message);
+  }
 }
 ```
 
 </CodeExample>
 
-For payments—our focus for this set of examples—the divergence only matters because we want to show the balances for the custodian account.
+when you invoke `runUnmuxedExample()`, you should see output similar to:
```

Comment:
nit, upper case 'When' , new sentence?

Reply:
follow-on thought is this example output shown could be elided for shorter guide overall, less cognitive reading required?